src/models/program.js

```diff
@@ -1,34 +1,35 @@
-const {
-  Model,
-} = require('sequelize');
-
-export default (sequelize, DataTypes) => {
-  class Program extends Model {
-    static associate(models) {
-      Program.belongsTo(models.Grant, { foreignKey: 'grantId', as: 'grant' });
-    }
-  }
-  Program.init({
-    id: {
-      type: DataTypes.INTEGER,
-      allowNull: false,
-      defaultValue: null,
-      primaryKey: true,
-      autoIncrement: false,
-    },
-    grantId: {
-      type: DataTypes.INTEGER,
-      allowNull: false,
-    },
-    programType: DataTypes.STRING,
-    startYear: DataTypes.STRING,
-    startDate: DataTypes.STRING,
-    endDate: DataTypes.STRING,
-    status: DataTypes.STRING,
-    name: DataTypes.STRING,
-  }, {
-    sequelize,
-    modelName: 'Program',
-  });
-  return Program;
-};
+const {

```

Comment:
weird, was this not saved with the LF endings? It makes it challenging to resolve merge conflicts (should there be any) when we have these different file endings floating around

Reply:
Not sure what I can do about this.